affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetectorPlugin.kt

```diff
@@ -74,9 +74,9 @@ class AffectedModuleDetectorPlugin : Plugin<Project> {
 
         rootProject.subprojects { project ->
             project.afterEvaluate {
-                val pluginIds = setOf("com.android.application", "com.android.library", "java-library", "kotlin")
+                val pluginIds = setOf("com.android.application", "com.android.library", "java", "kotlin")
```

Comment:
Thanks for the contribution!

Is there a reason we removed `java-library` from the `setOf`? Shouldn't we include both `java-library` and `java` ?

Reply:
`java-library` plugin applies `java` plugin. 
https://github.com/gradle/gradle/blob/master/subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaLibraryPlugin.java#L51

Reply:
Thanks for clarifying.   Once the CLA is signed, we can merge this.